docs/docs-content/troubleshooting/edge.md

```diff
@@ -414,3 +414,41 @@ issue, reset the SELinux context of the Kubelet environment variable to its defa
    ```shell
    systemctl restart kubelet
    ```
+
+## Scenario - Agent Mode Deployments CNI Folder Permission Issues
+
+Agent mode clusters that use PKX-E as the Kubernetes layer have the contents of the `/opt/cni/bin` folder set in
+correctly. This prevents the CNI from operating.
```

Comment:
This only affects CNI providers that **do not** run as root such as Cilium. For example, using Calico CNI does not require this workaround.